### PR TITLE
[Not for merge] Simon review

### PR DESCRIPTION
**Pull request by simontegg**
1. Install the a lint setup or use this one https://gist.github.com/simontegg/36d4b14207e1015b647eeeda2fbdc604

2. Fix the lint issues. See if the app still works. [run tests]

3. git fetch and checkout into the review branch. 
Inspect the comments and changes in your text editor and using the git diff below to see what has changed.

Either take a "muscle memory" approach - just manually type out the changes. Or pair up with somebody and walk through the comments and changes and discuss what you think is my rationale.

### CODE REVIEW

**Comment by heymartinadams** [comment]
Hi @simontegg,

Is there a reason you recommend that particular lint setup over the one currently in use?
https://github.com/swiftify-ai/swiftify/blob/master/.eslintrc

Thanks,
Martin

**Comment by simontegg** [comment]
Its mostly my personal preference. The jsx-a11y is good practice for accessiblity.

**Comment by heymartinadams** [comment]
Ok, thanks, @simontegg. I integrated jsx-a11y into the .eslintrc I’m using. The rest I’ve spent quite a bit of time configuring, so I’d like to keep it, unless you’d strongly advise against it.

File: public/components/app/actioninput.js [hunk 1]
```diff
@@ -7,42 +7,58 @@ class ActionInput extends React.Component {
...
+	handleChange(e) {
+	  this.setState({ value: e.target.value })
+	}
 
+  // put the MuiThemeProvider at the very top level component: App?
+  // its 'context' will flow down
+  // and matrial-ui components will know what to do
+  //
+  // is there a POST route setup?
+  // superflous unless react is also rendering this on the server.
```
Comment (heymartinadams) [inline public/components/app/actioninput.js]: What do you mean by 

> superflous unless react is also rendering this on the server

File: public/components/app/actioninput.js [hunk 2]
```diff
@@ -7,42 +7,58 @@ class ActionInput extends React.Component {
...
 	handleHover() {
+	  // dont need to write this as a closure
 		return (e) => {
 			this.setState({ elementBlurred: !this.state.elementBlurred })
 			return e.type === 'mouseenter' ? true : false
 		}
 	}
 
 	handleTouched() {
+	  // dont need to write this as a closure
```
Comment (heymartinadams) [inline public/components/app/actioninput.js]: Same here.

File: public/components/app/actioninput.js [hunk 3]
```diff
@@ -7,42 +7,58 @@ class ActionInput extends React.Component {
...
 		return (e) => {
 			let placeholder = e.type === 'focus' ? '' : 'Hi again :)'
 			this.setState({ placeholder })
 		}
 	}
+	
+	handleChange(e) {
```
Comment (heymartinadams) [inline public/components/app/actioninput.js]: This works no problem without being a closure.

**Pull request closed without merging**